reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -613,8 +613,8 @@ alter table SAM_ITEM_T add column MIN_SCORE double NULL;
 alter table SAM_PUBLISHEDITEM_T add column MIN_SCORE double NULL;
 
 -- KNL-1405 Don't have defaults for TIMESTAMP in SAKAI_SESSION
-ALTER TABLE SAKAI_SESSION MODIFY SESSION_START NULL;
-ALTER TABLE SAKAI_SESSION MODIFY SESSION_END NULL;
+ALTER TABLE SAKAI_SESSION MODIFY SESSION_START TIMESTAMP NULL;
+ALTER TABLE SAKAI_SESSION MODIFY SESSION_END TIMESTAMP NULL;
```

Comment:
On MySQL 5.5.46 without the column type specification I get an error of:

    Error Code: 1064. You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NULL' at line 1